proto/cacheclient.proto

```diff
@@ -451,9 +451,8 @@ message _SortedSetPutResponse {}
 
 message _SortedSetFetchRequest {
   enum Order {
-    UNKNOWN = 0;
-    ASCENDING = 1;
-    DESCENDING = 2;
+    ASCENDING = 0;
+    DESCENDING = 1;
   }
```

Comment:
This renumbering will break any existing code's compatibility with the server. If there's nothing using this yet that's fine, but if there's any client using this it needs to be accounted for - is this currently unused?

Reply:
Correct this is currently unused, this is only for the new sorted sets api which is unimplemented. In protobuf, enums need to start with 0, so if we want to remove `UNKNOWN` we need to renumber the enums